supabase/config.toml

```diff
@@ -74,10 +74,9 @@ file_size_limit = "50MiB"
 enabled = true
 
 # Uncomment to configure local storage buckets
-# [storage.buckets.images]
-# public = false
-# file_size_limit = "50MiB"
-# allowed_mime_types = ["image/png", "image/jpeg"]
+[storage.buckets.postgres-new-storage]
```

Comment:
How important is it to have the `postgres-new` namespace here? I could be wrong but I don't think storage bucket names need to be globally unique. Might be nice to just call this "certs" or something more generic so that we have an easier time later renaming.

Reply:
This bucket will also potentially contains pgdata archives (easier to have one bucket than multiple).
Maybe just "storage"?

Reply:
Yep that works 👍

Reply:
Done!